processor/schematransformerprocessor/README.md

```diff
@@ -0,0 +1,25 @@
+# Schema Transformer Processor
+
+🚧 _Currently under development, subject to change_ 🚧
+
+Supported Pipelines: traces, metrics, logs
+
+The _Schema Transformer Processor_ is used to convert existing telemetry data or signals to the most recent targeted version of the semantic convention defined as part of the configuration. The latest schema translation file will be bundled as part of the collector build to avoiding having to fetch it at runtime. 
+
+Specification Reference: [here](https://opentelemetry.io/docs/reference/specification/schemas/overview/)
+
+
+# Schema Definitions
+All of the published schema transformation are found by `https://opentelemetry.io/schemas/`
+
+# Example
+
+```yaml
+processors:
+    schema_transformer:
+        schema:
+            url: https://opentelemetry.io/schemas/1.9.0
+            target: https://opentelemtry.io/schemas/1.5.0
```

Comment:
The settings names don't match what is `Config` struct. Also, the purpose of `url` is not clear.

Reply:
This is me being terrible at documentation, let me go back and chase this up :)